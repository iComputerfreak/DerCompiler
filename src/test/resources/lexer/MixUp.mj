this ; is , a [ mixup with null or 0 sense _test
123 switch /*123*/
