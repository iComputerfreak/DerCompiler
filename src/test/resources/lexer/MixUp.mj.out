this
;
identifier is
,
identifier a
[
identifier mixup
identifier with
null
identifier or
integer literal 0
identifier sense
identifier _test
integer literal 123
switch
EOF
